source/distributed/fully_distributed_tria_util.cc

```diff
@@ -444,6 +449,26 @@ namespace parallel
                     return false;
                   };
 
+                std::vector<bool> vertices_owned_by_locally_owned_cells_strong(
```

Comment:
You might want to add a comment to this variable and to the lambda just below that explains what "strong" refers to.